cmd/healthbot/config.go

```diff
@@ -35,6 +35,10 @@ type ChainConfig struct {
 		ReceiptTimeout string `default:"20s"`
 		Tablename      string `default:""`
 	}
+	OverrideClient struct {
```

Comment:
New Healthbot config to override the [config of chains of the client](https://github.com/tablelandnetwork/go-tableland/blob/ad7d0a5913f7cca7f396e346c5a2f7a5bb7e57ff/pkg/client/chains.go). This is needed because a `staging` network is not official and does not exist as part of the client configuration